packages/gatsby-theme-carbon/src/components/LeftNav/LeftNav.module.scss

```diff
@@ -11,6 +11,13 @@
   color: $ui-05;
 }
 
+.divider {
+  margin: $spacing-03 $spacing-05;
+  border-color: var(--cds-ui-03, #e0e0e0);
```

Comment:
why the variable? why not just use `$ui-03`?

Reply:
@sstrubberg Hmm..I wanted to do so, but then I looked at how the divider that separates resources links from nav items and this was the code for them.

LeftNav.module.css: 
![image](https://user-images.githubusercontent.com/24867712/85593711-f2c6ae00-b64f-11ea-8372-6ab8e30b8cc0.png)